docs/development/upgrading-chrome.md

```diff
@@ -1,47 +1,65 @@
-# Upgrading Chrome Checklist
+# Upgrading Chromium Workflow
 
 This document is meant to serve as an overview of what steps are needed
-on each Chrome upgrade in Electron.
+on each Chromium upgrade in Electron.
 
-These are things to do in addition to updating the Electron code for any
-Chrome/Node API changes.
+## Update libchromiumcontent (a.k.a. libcc)
+
+- Clone the repo:
+```
+git clone git@github.com:electron/libchromiumcontent.git
+cd libchromiumcontent
+```
+- Run bootstrap script to init and sync git submodules:
+```
+./script/bootstrap -v
+```
+- Update `VERSION` file to correspond to Chromium version.
+- Run `script/update`, it will probably fail applying patches.
+- Fix failing patches. `script/patch.py` might help.
+  - Don't forget to fix patches in the `patches-mas/` folder.
+- Build libcc:
+```
+./script/build
+```
+- Create dist folders which will be used by electron:
+```
+./script/create-dist --no_zip
```

Comment:
I'm not sure if it's a good idea to use so detailed commands in this guide. I mean it might be hard to keep them up-to-date all the time. Maybe it would make more sense to say _what_ should be done instead of _how_?, and provide links to libcc/Electron build docs? @tonyganch, what do you think?